jackal/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "jackal",
+  "chain_id": "jackal-1",
+  "website": "https://jackalprotocol.com",
+  "pretty_name": "Jackal",
+  "status": "live",
+  "network_type": "mainnet",
+  "bech32_prefix": "jkl",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1432
in progress thank you